Mathlib/Algebra/Algebra/Tower.lean

```diff
@@ -297,6 +297,10 @@ variable [Module R S] [Module S A] [Module R A] [IsScalarTower R S A]
 
 open IsScalarTower
 
+theorem span_le_restrictScalars_span (t : Set A) : span R t ≤ (span S t).restrictScalars R :=
```

Comment:
Is this [`Submodule.span_le_restrictScalars`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/LinearAlgebra/Span.html#Submodule.span_le_restrictScalars)?

Reply:
Indeed, thanks! I just wanted to refactor the two lemmas at the bottom using the main result `span_smul_of_span_eq_top`, and hastily proved this lemma by hand without looking further whether it already existed.